bin/get-amy.py

```diff
@@ -37,11 +44,39 @@ def main(amy_url, output_file, tags):
         'airports' : fetch_info(amy_url, 'export/instructors.yaml')
     }
 
+    if all_:
+        # Fetch workshops that have all of the specified tags.
+        unique_workshops = fetch_info(amy_url, 'events/published.yaml',
+                                      tags=tags)
+    else:
+        # Fetch workshops for each tag. This is required because by AMY treats
+        # with AND operation, while we want to have OR: "get me workshops that
+        # have either SWC tag or DC tag (or both)".
+        workshops = []
+        for tag in tags:
+            workshops.extend(fetch_info(amy_url, 'events/published.yaml',
+                                        tags=[tag]))
```

Comment:
Hm. OK, please add a huge note saying "this will go away when AMY is fixed to handle both AND and OR" and then file the issue to come back and fix it when that happens.  Otherwise, this code will still be here long after AMY is fixed :-)